### ISSUE TITLE
Crash on Raspberry PI 4

### ISSUE THREAD

**Issue by pupaxxo** [fyne-io/fyne#2603]
### Describe the bug:

Crash when launching app on Raspberry PI 4

```
panic: unaligned 64-bit atomic operation

goroutine 6 [running, locked to thread]:
runtime/internal/atomic.panicUnaligned()
	/tmp/golang/go/src/runtime/internal/atomic/unaligned.go:8 +0x24
runtime/internal/atomic.Load64(0x14a205c)
	/tmp/golang/go/src/runtime/internal/atomic/atomic_arm.s:286 +0x14
fyne.io/fyne/v2/internal/driver/common.(*Canvas).FreeDirtyTextures(0x14a2000)
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/common/canvas.go:205 +0x38
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).repaintWindow.func1()
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/loop.go:171 +0x78
fyne.io/fyne/v2/internal/driver/glfw.(*window).RunWithContext(0x149e000, 0x1440f2c)
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/window.go:1336 +0x48
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).repaintWindow(0x170bd80, 0x149e000)
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/loop.go:165 +0x50
fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).startDrawThread.func1(0x170bd80, 0x1421a80, 0x170be80)
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/loop.go:228 +0x2bc
created by fyne.io/fyne/v2/internal/driver/glfw.(*gLDriver).startDrawThread
	/home/pi/go/pkg/mod/fyne.io/fyne/v2@v2.1.1/internal/driver/glfw/loop.go:192 +0xa0
exit status 2
```

**Comment by Jacalz** [comment]
In the future, please fill out the entire issue template. However, this time it’s fine because I am pretty sure that this has already been fixed with https://github.com/fyne-io/fyne/commit/5ace18d964f0a8cba2766659fd31fb95f34d73de. Could you please test the `release/v2.1.x` branch (or the above commit)?

**Comment by lugu** [comment]
I have the same problem on a 32 bits Android phone using the develop branch.

**Comment by andydotxyz** [comment]
This should be resolved on `release/v2.1.x` branch by 5ace18d964f0a8cba2766659fd31fb95f34d73de
